test/unit/services/policies/policies_list_service_test.rb

```diff
@@ -23,16 +25,43 @@ def setup
   }.freeze
 
   test 'call with error response on gateway side' do
-    stub_request(:get, "https://apicast-staging.proda.example.com/policies")
+    stub_request(:get, APICAST_REGISTRY_URL)
```

Comment:
Good 👍 But since this line is done all over the place, maybe it would be nice to extract it to a method such as `stub_apicast_request` 😄